app/src/main/java/com/mensinator/app/SettingsDialog.kt

```diff
@@ -212,34 +212,43 @@ fun SettingsDialog(onSwitchProtectionScreen: (Boolean) -> Unit) {
                                             )
                                         }
                                     }
-                                    DropdownMenu(
-                                        modifier = Modifier
-                                            .width(50.dp)
-                                            .height(menuHeight),
-                                        expanded = expanded,
-                                        onDismissRequest = { expanded = false }
+                                    MaterialTheme(
```

Comment:
We should not create a new MaterialTheme for each dialog. There should only be one `MaterialTheme` defined for the entire app. Instead, we should define & use custom shapes and use them at `MensinatorTheme`. The dialogs will then automatically pick up the shape definitions.

Theme.kt would look like this:

```
    val shapes = Shapes(
        extraSmall = RoundedCornerShape(16.dp)
    )

    MaterialTheme(
        colorScheme = colorScheme,
        shapes = shapes,
        typography = Typography,
        content = content
    )
```

Reply:
Please add this, revert the theme-related changes in here, and also remove the MaterialTheme() at `ManageSymptomDialog.kt` :)

Reply:
thats so true, thanks!

Reply:
thats so true thanks!